brianlong/www.validators.app
# 187469924-gems-upgrade

#### What's this PR do?
- as in title

#### How should this be manually tested?
- bundle install
- run tests
- go through the website and check if everything works as expected

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187469924)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
